### PR TITLE
Created a test.md file to test the review process

### PR DESCRIPTION
**Pull request by hongsups**
Regarding our discussion today about using GitHub to review our work product, I've created a short markdown example to simulate a review process. This is based on our officer-involved shooting report and I just copied and pasted some parts of it. I've assigned the reviewers (people who were at the meeting today and Nick) to this pull request. So feel free to make comments (anything really) on the `test.md` file (or even on the image file; ignore the .eps image file).

Here are several things that I noticed:
- I think I misspoke about this process when @lukasbayern18 asked me about the rendering of Latex. At the conference I attended, they had a specific app that rendered latex because GitHub markdown cannot render Latex. I did some digging and it seems like there are workarounds but I haven't tried any. I think Netlify has some markdown apps but I haven't tried that either. So this is something to discuss when we need to render latex (especially formulas).
- In terms of writing plain text and incorporating code, I think this is very straightforward. However, including images and doing more complicated things like one would do with html and css, may not be the best use case for this (we need some external code that helps this process).
- The way github and Netlify render markdown might be different. You can write a document in github website and then you can see the preview of your markdown document when it's rendered on the website, but this does not necessarily mean it will be rendered in the same way in Netlify. So we will need to test this.
- So for this quick experiment, in summary: 
  - This process can be very useful to build a transparent and detailed review process.
  - If you're an author, you need to learn some markdown or rst syntax. If you're a reviewer, you don't have to.
  - It can be challenging to incorporate sophisticated web design stuff (and probably latex) in markdown. We will need additional step.
  - Plain text, code, and other files that we can post here (images and such) can be still reviewed.
  - Source code and work product (text, image, etc.) can exist in one place.

Let me know what you guys think.

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+|**Officers Shot**  |Officers who are either injured or killed |
+|**Officer Deaths** |Officers who are killed                   |
+
+The details of data acquisition and data preprocessing can be found on our [website](https://texasjusticeinitiative.org/) and our [public GitHub repository](https://github.com/texas-justice-initiative/).
+
+## 1. Comparison between the OAG annual reports and TJI multi-year datasets
+The total number of civilians shot by officers in Texas from 2016 to 2019 is identical in the OAG reports and TJI datasets (697 incidents). However, the totals from individual years are slightly different (Fig. 1).
+
+<p align="center">
+  <img src="Figures/Test/Figure_1.jpg" width=640 alt="Fig.1. Number of civilians shot in officer-involved shootings from 2016 to 2019 in the OAG reports and TJI datasets">
```
Comment (nholden) [inline test.md:30]: This image isn't loading for me when I view the rendered markdown, but I *think* this change might fix it.

```suggestion
  <img src="./Figures/Test/Figure_1.jpg" width=640 alt="Fig.1. Number of civilians shot in officer-involved shootings from 2016 to 2019 in the OAG reports and TJI datasets">
```

File: test.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# OFFICER-INVOLVED SHOOTINGS IN TEXAS: 2016-2019
```
Comment (nholden) [inline test.md:1]: I would use title-case here and let the heading levels show the hierarchy of the document.

```suggestion
# Officer-Involved Shootings in Texas: 2016-2019
```

File: test.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+*The Texas Justice Initiative (TJI) is a nonprofit organization that collects, analyzes, publishes and provides oversight for criminal justice data throughout Texas. This “Officer-Involved Shootings in Texas" report analyzes data from reports filed for shootings over a four-year period. The goal of this report is to identify patterns and make comparisons of officer and civilian injuries and deaths during officer-involved (OIS) shooting incidents across a variety of variables.*
+
+# I. Executive Summary
```
Comment (nholden) [inline test.md:5]:
```suggestion
## I. Executive Summary
```

File: test.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+Texas law requires the Office of the Attorney General (OAG) to collect reports on all officer-involved shootings throughout Texas and then provide the summary of the individual reports in “The Peace Officer Involved Shooting Annual Reports.” The OAG digitally publishes these annual reports, which are extremely high-level summaries, with no findings or analysis of trends or intersectionality. Texas Justice Initiative (TJI) reviewed the officer-involved shooting incidents data from law enforcement agencies in Texas between 2016 and 2019. This TJI publication improves upon the OAG’s annual reports by providing in-depth analyses with more granular information. **This report consists of two parts**:
+- The **Data Summary** section provides an overview of the incidents as intersectional information such as the total number of civilian severities per county and per race demographics.
+- The **Data Insight** section provides TJI’s independent analysis of the officer-involved shootings, including notable patterns in the data.
+
+-----
+# II. Data Summary
```
Comment (nholden) [inline test.md:11]:
```suggestion
## II. Data Summary
```

File: test.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+|Terms              |Corresponding sub population in the data  |
+|-------------------|------------------------------------------|
+|**Civilians Shot** |Civilians who are either injured or killed|
+|**Civilian Deaths**|Civilians who are killed                  |
+|**Officers Shot**  |Officers who are either injured or killed |
+|**Officer Deaths** |Officers who are killed                   |
+
+The details of data acquisition and data preprocessing can be found on our [website](https://texasjusticeinitiative.org/) and our [public GitHub repository](https://github.com/texas-justice-initiative/).
+
+## 1. Comparison between the OAG annual reports and TJI multi-year datasets
```
Comment (nholden) [inline test.md:26]:
```suggestion
### 1. Comparison between the OAG annual reports and TJI multi-year datasets
```

File: test.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+The total number of civilians shot by officers in Texas from 2016 to 2019 is identical in the OAG reports and TJI datasets (697 incidents). However, the totals from individual years are slightly different (Fig. 1).
+
+<p align="center">
+  <img src="Figures/Test/Figure_1.jpg" width=640 alt="Fig.1. Number of civilians shot in officer-involved shootings from 2016 to 2019 in the OAG reports and TJI datasets">
+</p>
+<p align="center">  
+  <em>Fig.1. Number of civilians shot in officer-involved shootings from 2016 to 2019 in the OAG reports and TJI datasets</em>
+</p>
+
+# IV. Conclusions
```
Comment (nholden) [inline test.md:36]:
```suggestion
## III. Conclusions
```

File: test.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
+# OFFICER-INVOLVED SHOOTINGS IN TEXAS: 2016-2019
+
+*The Texas Justice Initiative (TJI) is a nonprofit organization that collects, analyzes, publishes and provides oversight for criminal justice data throughout Texas. This “Officer-Involved Shootings in Texas" report analyzes data from reports filed for shootings over a four-year period. The goal of this report is to identify patterns and make comparisons of officer and civilian injuries and deaths during officer-involved (OIS) shooting incidents across a variety of variables.*
```
Comment (nholden) [inline test.md:3]:
```suggestion
*The Texas Justice Initiative (TJI) is a nonprofit organization that collects, analyzes, publishes and provides oversight for criminal justice data throughout Texas. “This Officer-Involved Shootings in Texas” report analyzes data from reports filed for shootings over a four-year period. The goal of this report is to identify patterns and make comparisons of officer and civilian injuries and deaths during officer-involved shooting (OIS) incidents across a variety of variables.*
```

File: test.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+- The **Data Insight** section provides TJI’s independent analysis of the officer-involved shootings, including notable patterns in the data.
+
+-----
+# II. Data Summary
+
+The OAG annual reports, and this TJI four-year analysis report, summarize officer-involved shooting reports submitted to the OAG by local law enforcement agencies. The reports reflect shootings of and by law enforcement officers that took place in Texas from 2016 to 2019.
+
+Every record in the officer-involved shooting data represents either a civilian or an officer who was shot. This means every person in the dataset was shot and harmed. Currently the data represents the severity of the shooting as binary information: **injury** or **death**. For the majority of the analyses, we focus on two populations: **officers** and **civilians**. We use the following terminology through the report:
+
+|Terms              |Corresponding sub population in the data  |
```
Comment (nholden) [inline test.md:17]:
```suggestion
|Terms              |Corresponding sub-population in the data  |
```

File: test.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+|-------------------|------------------------------------------|
+|**Civilians Shot** |Civilians who are either injured or killed|
+|**Civilian Deaths**|Civilians who are killed                  |
+|**Officers Shot**  |Officers who are either injured or killed |
+|**Officer Deaths** |Officers who are killed                   |
+
+The details of data acquisition and data preprocessing can be found on our [website](https://texasjusticeinitiative.org/) and our [public GitHub repository](https://github.com/texas-justice-initiative/).
```
Comment (nholden) [inline test.md:24]: Should we link to a specific repo here?

**Review by nholden — COMMENTED** [review]
👋 Thanks for pitching this approach for a review process, @hongsups! 💖

I think this approach makes sense to me if both of the following are true:

1. The process is intuitive and relatively painless for reviewers
2. Markdown formatting is useful for us

I'm probably not the best person to evaluate 1, since I ❤️ GitHub and markdown. From the few nit-picky comments I left here, I would say it was a decent experience for me. Inline feedback and suggestions seem like a good way to discuss individual tweaks, and comments on the PR make sense for holistic feedback. It was a little bit of a pain to jump back and forth between the raw version and the rendered version, and leaving comments or suggestions on long paragraphs (which are a single line in markdown) was a little unwieldy, but overall it was a good experience.

On 2, Markdown is super helpful if we're targeting the website since that's how most of our content is stored. If we're targeting a PDF report that we want to generate via LaTeX or a Google Doc, it's hard to say if Markdown is useful or just an extra step.

I think this approach is better than commenting on a Google Doc in terms of discoverability of drafts and feedback and in terms of tracking changes. However, if those couple of things above don't pan out, we may want to consider commenting on a Google Doc, or another approach altogether.

File: test.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+</p>
+
+# IV. Conclusions
+Texas Justice Initiative’s analyses of officer-involved shootings in Texas over four years (2016-2019) reveals several notable patterns. Among our main findings:
+
+1. Shootings of civilians and their subsequent deaths caused by officers have been increasing over the four years (Figs. 1 and 15)
+2. Officers were twice as likely to survive shootings than civilians who are shot by officers (Fig. 16)
+3. As one might expect, the majority of officer-involved shootings in Texas occurred in the state's five most populous counties. DPS shooting reports also comprise a large portion of those filed every year (Fig. 14)
+4. Black individuals were overrepresented in the five most populous counties compared to the general population (Fig. 18)
+5. Among those shot and killed by Texas law enforcement, young Black men ages 15-24 were over-represented at a rate of almost three  times when compared to the general population (Fig. 20)
```
Comment (EvaRuthM) [inline test.md:43]:
```suggestion
5. Among those people shot and killed by Texas law enforcement, young Black men ages 15-24 were over-represented at a rate of almost three  times when compared to the general population (Fig. 20)
```

Comment (hongsups) [reply test.md:43]: That's a great idea Eva!

**Comment by hongsups** [comment]
Closing the PR since we are migrating to CMS.

**Pull request closed without merging**